session/handler.go

```diff
@@ -459,6 +485,92 @@ func (h *Handler) IsAuthenticated(wrap httprouter.Handle, onUnauthenticated http
 	}
 }
 
+// swagger:parameters adminSessionRefresh
+// nolint:deadcode,unused
+type adminSessionRefresh struct {
+	// ID is the session's ID.
+	//
+	// required: true
+	// in: path
+	ID string `json:"id"`
+}
+
+// swagger:route PATCH /sessions/refresh/{id} v0alpha2 adminSessionRefresh
```

Comment:
We try to have structure with REST resource model, so this would be:

```suggestion
// swagger:route PATCH /sessions/{id}/refresh v0alpha2 adminRefreshSession
```

or


```suggestion
// swagger:route PATCH /sessions/{id}?refresh=true v0alpha2 adminRefreshSession
```